backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Cursor.java

```diff
@@ -105,10 +107,13 @@ static void disposeSystemCursors () {
 
 	static void setSystemCursor (long windowHandle, SystemCursor systemCursor) {
 		if (systemCursor == SystemCursor.None) {
+			if (inputModeBeforeNoneCursor == -1) inputModeBeforeNoneCursor = GLFW.glfwGetInputMode(windowHandle, GLFW.GLFW_CURSOR);
 			GLFW.glfwSetInputMode(windowHandle, GLFW.GLFW_CURSOR, GLFW.GLFW_CURSOR_HIDDEN);
 			return;
-		} else {
-			GLFW.glfwSetInputMode(windowHandle, GLFW.GLFW_CURSOR, GLFW.GLFW_CURSOR_NORMAL);
+		} else if (inputModeBeforeNoneCursor != -1) {
+			GLFW.glfwSetInputMode(windowHandle, GLFW.GLFW_CURSOR,
+				inputModeBeforeNoneCursor == -1 ? GLFW.GLFW_CURSOR_NORMAL : inputModeBeforeNoneCursor);
+			inputModeBeforeNoneCursor = -1;
 		}
```

Comment:
That looks odd, unless I am missing something? The`else if (...)` should just be an `else`, or the `... ? ...` part can be simplified!?

Reply:
I agree with the fix otherwise! 👍

Reply:
Yes, my bad.